components/server/src/orgs/organization-service.ts

```diff
@@ -145,6 +147,19 @@ export class OrganizationService {
     }
 
     async createOrganization(userId: string, name: string): Promise<Organization> {
+        // TODO(gpl): Should we use the authorization layer to make this decision?
```

Comment:
Because it's static, we could also make it a relation. But there's also a way to attach data to a request, and "dynamic" rules that can decide based on that data.

But anyway, this is more speculative, as I don't see it being worth the effort to pull off right now.